update-translations.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+CWD=`pwd`
+DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd $DIRECTORY
+
+function log() {
+  if [[ -n $1 ]];then
+    printf "\033[0;36m\n$1\n\033[0m"
```

Comment:
You might be able to use `tput` to set the colours:

```bash

# Text color variables
txtund=$(tput sgr 0 1)    # Underline
txtbld=$(tput bold)       # Bold
txtred=$(tput setaf 1)    # Red
txtgrn=$(tput setaf 2)    # Green
txtylw=$(tput setaf 3)    # Yellow
txtblu=$(tput setaf 4)    # Blue
txtpur=$(tput setaf 5)    # Purple
txtcyn=$(tput setaf 6)    # Cyan
txtwht=$(tput setaf 7)    # White
txtrst=$(tput sgr0)       # Text reset
```

(from [SO](https://stackoverflow.com/a/2495357))